GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -27,6 +27,26 @@
 
 @implementation GIDVerifyAccountDetail
 
+- (instancetype)initWithConfig:(GIDConfiguration *)configuration {
```

Comment:
Please add a test for this.

Reply:
Done.

Reply:
Yes, please add tests where the `GIDConfiguration` is passed in and not created automatically (via a fake).

Reply:
Added, please let me know if I missed any.